### ISSUE TITLE
Scroll Issue

### ISSUE THREAD

**Issue by DhimantB** [Silvere112/ngx-load#1]
The loader is moving with the page scroll, which is expected behavior.

When I applied it in below code, scrolling is having an issue

```
<div class="has-scroll">
  <div [loaderCdkOverlay]></div>
</div>
```

Here the main div has internal scrolling. When it scrolls, the loader is not moving.

So I noticed the cdk overlay is being attached to the document body, I think that's the issue.

I am guessing if it would have being attached the element itself instead of body then I believe it will work

**Comment by Silvere112** [comment]
You are right under the wood the library uses the overlay provided by @angular/cdk which is attached to the body itself however this should not be an issue because the loader should be repositioned. https://github.com/Silvere112/ngx-load/blob/d4f23495d67b11c35989e2893a55a8b8e9d75cab/projects/load/src/lib/overlay/overlay-loader.directive.ts#L55

I tried to produce the same scenario without success. Here is what I got: 

![gif-scroll](https://github.com/Silvere112/ngx-load/assets/26688374/c0e2e0e0-cb20-4f6a-a570-f4a4ae61fae7)

The code is available here: https://github.com/Silvere112/ngx-load-examples/commit/f65b0e35534fab5c3b2a09fa2c5f5509c90325b7

Can you provide a real case example? Do you use the property overflow: scroll; ?

**Comment by Silvere112** [comment]
I may understood your issue. Can you try to add the directive cdkScrollable to the "has-scroll" container and import the ScrollingModule from Angular CDK to make it work ?
